.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!-- Thank you for contributing! Please make sure that your code changes
+are covered with tests. In case of new features or big changes remember
+to adjust the documentation.
+
+In case of an existing issue, reference it using one of the following:
+
+closes: #ISSUE
+related: #ISSUE
+
+How to write a good git commit message:
+http://chris.beams.io/posts/git-commit/
```

Comment:
saw the `http` and was just curious
```
curl -I http://chris.beams.io/posts/git-commit/
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Tue, 13 Dec 2022 16:40:06 GMT
Content-Type: text/html
Content-Length: 162
Connection: keep-alive
Location: https://cbea.ms/posts/git-commit/
```

Reply:
I was drawing some inspiration from https://raw.githubusercontent.com/apache/airflow/main/.github/PULL_REQUEST_TEMPLATE.md